### ISSUE TITLE
Loading a pretrained checkpoint of a QAT model (Resnet 50)

### ISSUE THREAD

**Issue by peri044** [tensorflow/models#8658]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [x] I checked to make sure that this issue has not been filed already.

I'm using TF 2.2.0-dev20200506

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/vision/image_classification/resnet

## 2. Describe the bug

I trained a Resnet-50 model with quantization aware training. However when I try to load the final checkpoint to export to a saved model, I see "Unresolved object in checkpoint" issues.  Please check the complete log [here ](https://gist.github.com/peri044/d78c6ff6f96db1ac0e899738a97b293b)
I perform QAT on the resnet by adding the following snippet of code in resnet_runnable.py [here](https://github.com/tensorflow/models/blob/master/official/vision/image_classification/resnet/resnet_runnable.py#L76)

`import tensorflow_model_optimization as tfmot` 
` quantize_model = tfmot.quantization.keras.quantize_model`
` self.model = quantize_model(self.model)`

I'm trying to export the saved model and face the above unresolved object in checkpoint error when loading weights https://gist.github.com/peri044/6decce75fb94c3ae3a2dee20eb5dd7a2#file-load-py-L47

Is there something obvious that I'm missing here ? Please advise. Thank you !!

## 3. Additional context

My checkpoint has the variable names under layer_with_weights and I see the kernel_min and kernel_max values for the quantization nodes. But the variables TF is looking for are `quant_bn2a_branch1` etc. Is this the right way for topological loading?

**Comment by Ruomei** [comment]
@peri044 
Just curious, what accuracy numbers have you got for the resnet_vanilla and resnet_qat here?

**Comment by peri044** [comment]
The accuracy drop was around 0.2 in my experiments. Pretrained RN50: 76.4 and the QAT version 76.2. I used quantize_and_dequantize operations instead of fake_quant_with_min_max_var.
